project/src/main/webapp/WEB-INF/web.xml

```diff
@@ -104,4 +104,16 @@
         <url-pattern>/listAssignmentSubmissions</url-pattern>
     </servlet-mapping>
 
+    <!-- Manage Box (Outlines) -->
+    
+    <servlet>
+        <servlet-name>ManageBoxServlet</servlet-name>
+        <servlet-class>com.google.autograder.servlets.ManageBoxServlet</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>ManageBoxServlet</servlet-name>
+        <url-pattern>/ManageBoxServlet</url-pattern>
```

Comment:
The url-pattern needs to match what's in your js, which is currently /ManageBox.

Reply:
my bad